### ISSUE TITLE
Rule: When a query includes a computed column in the SELECT list, warn if no column alias was defined

### ISSUE THREAD

**Issue by barrywhart** [alanmcruickshank/sqlfluff#12]
This makes the code and the query results more self documenting. It's especially important in some data pipelines, where SQL queries are often stored in new tables even without an explicit `CREATE TABLE AS` statement.

**Comment by alanmcruickshank** [comment]
I think this is a great suggestion. 👍

**Comment by alanmcruickshank** [comment]
@barrywhart - I've just published 0.2.1 which includes this feature (and some other things too). Can you check it out and see how it works?

**Comment by barrywhart** [comment]
Yes, this sounds great! I'm on holiday this week but return tomorrow. Our team has added a new member recently so I hope to have more time to test and contribute. 🤞

**Comment by barrywhart** [comment]
I'm working on getting sqlfluff integrated into our CI builds -- you may have seen an issue or two I created already.

It looks like it's working, as I see this in the output from one of our SQL files.
```
L: 150 | P:   1 | L011 | Implicit aliasing of table not allowed. Use explicit `AS` clause.
```

**Comment by barrywhart** [comment]
Oh wait, my earlier comment was incorrect -- there I was looking at table aliases.

I tried to test this change, and I'm not seeing the expected behavior. For example, I expected to see a warning for this query:
```
SELECT
    COUNT(1)
FROM
    p.d.t
```

**Comment by barrywhart** [comment]
@alanmcruickshank: Based on the comment above, can we reopen this issue?
